app/services/metric_service.rb

```diff
@@ -0,0 +1,58 @@
+class MetricService
```

Comment:
I would call this `MetricsService`, but don't think it matters much.

Reply:
renamed